ui/src/views/settings/SettingsQueue.vue

```diff
@@ -0,0 +1,321 @@
+    UtilService,
+    IconService,
+    QueryParamService,
+    PageTitleService,
+  ],
+  pageTitle() {
+    return this.$t("settings_mailboxes.title") + " - " + this.appName;
+  },
+  data() {
+    return {
+      q: {
+        page: "settingsQueue",
+      },
+      urlCheckInterval: null,
+      maximal_queue_lifetime: "120",
+      loading: {
+        getQueueSettings: false,
+        setQueueSettings: false,
+      },
+      error: {
+        getQueueSettings: "",
+        setQueueSettings: "",
+        maximal_queue_lifetime: "",
+      },
+    };
+  },
+  computed: {
+    ...mapState(["core", "appName", "instanceName"]),
+  },
+  watch: {},
+  beforeRouteEnter(to, from, next) {
+    next((vm) => {
+      vm.watchQueryData(vm);
+      vm.urlCheckInterval = vm.initUrlBindingForApp(vm, vm.q.page);
+    });
+  },
+  beforeRouteLeave(to, from, next) {
+    clearInterval(this.urlCheckInterval);
+    next();
+  },
+  created() {
+    this.getQueueSettings();
+  },
+  methods: {
+    goToSettings() {
+      this.goToAppPage(this.instanceName, "settings");
+    },
+    async getQueueSettings() {
+      this.loading.getQueueSettings = true;
+      this.error.getQueueSettings = "";
+      const taskAction = "get-queue-settings";
+      const eventId = this.getUuid();
+
+      // register to task error
+      this.core.$root.$once(
+        `${taskAction}-aborted-${eventId}`,
+        this.getQueueSettingsAborted
+      );
+
+      // register to task completion
+      this.core.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.getQueueSettingsCompleted
+      );
+
+      const res = await to(
+        this.createModuleTaskForApp(this.instanceName, {
+          action: taskAction,
+          extra: {
+            title: this.$t("action." + taskAction),
+            isNotificationHidden: true,
+            eventId,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.getQueueSettings = this.getErrorMessage(err);
+        this.loading.getQueueSettings = false;
+        return;
+      }
+    },
+    getQueueSettingsAborted(taskResult, taskContext) {
+      console.error(`${taskContext.action} aborted`, taskResult);
+      this.error.getQueueSettings = this.$t("error.generic_error");
+      this.loading.getQueueSettings = false;
+    },
+    getQueueSettingsCompleted(taskContext, taskResult) {
+      this.loading.getQueueSettings = false;
+      this.maximal_queue_lifetime = taskResult.output.maximal_queue_lifetime;
+    },
+    validateSetQueueSettings() {
+      this.clearErrors();
+      let isValidationOk = true;
+
+      if (!this.maximal_queue_lifetime) {
+        this.error.maximal_queue_lifetime = this.$t("common.required");
+
+        if (isValidationOk) {
+          this.focusElement("maximal_queue_lifetime");
+          isValidationOk = false;
+        }
+      }
+      return isValidationOk;
```

Comment:
Let's add minimum and maximum validation so that error validations always have the same look&feel (red border and message)